src/routes/privacy/index.tsx

```diff
@@ -99,25 +99,21 @@ export default component$(() => {
       <Text tag="h3">Types of Data Collected</Text>
       <Text tag="h4">Personal Data</Text>
       <Text>
-        While using Our Services, We may ask You to provide Us with certain
-        personally identifiable information that can be used to contact or
-        identify You. Personally identifiable information may include, but is
-        not limited to:
+        During the use of our services, we may request that you provide us with
```

Comment:
Let's keep capitalization for "Our", "Services", "You", "Us" etc, since they are defined to represent different things at the start of the document.

Reply:
I also don't know if we need the part where we may ask for information? We don't do that, and I don't really think it's something we will do in the near future either. Should probably be something like "While using Our Services, we may collect the following information: ..."

What do you think?